zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -229,67 +219,53 @@ Wallet developers and operators should monitor the Zcash network for rapid
 growth in transaction rates.
 
 
-Transaction relaying
---------------------
-
-zcashd, and potentially other node implementations, implement fee-based
-restrictions on relaying of mempool transactions. Nodes that normally relay
-transactions are expected to do so for transactions that pay at least the
-conventional fee, unless there are other reasons not to do so for robustness
-or denial-of-service mitigation.
```

Comment:
This was a duplicate of https://github.com/nighthawk24/zips/pull/2/files#diff-25c7a2e2882ef10e0dddc86b7a276339eac6e57993a5f30866a773c4929df4b9R105-R112